ale_linters/yaml/actionlint.vim

```diff
@@ -0,0 +1,11 @@
+" Author: bretello <bretello@distruzione.org>
+
+call ale#Set('yaml_actionlint_executable', 'actionlint')
+call ale#Set('yaml_actionlint_options', '--no-color --oneline')
```

Comment:
I believe these two options are required for the handler to work properly?  If users set different options the handler pattern can break and the linter won't work.  If this is the case then these two options should be set in the command no matter what yaml_actionlint_options is set to.

Reply:
Got rid of actionlint options. For the record, the pattern can be customized calling actionlint with the following syntax:
```actionlint -format '{{range $err := .}}{{$err.Filepath}}{{$err.Line}}:{{$err.Column}}: [{{$err.Kind}}] {{$err.Message}}\n{{end}}'```